### PR TITLE
Make the equip sphere emissive

### PR DESCRIPTION
**Pull request by davidkelly**
The equip sphere was difficult to see in low ambient light conditions, per https://highfidelity.fogbugz.com/f/cases/1264#.  So, I switched to use one of Alan's models for an equip sphere, which is emissive and semi-transparent.  Now visible in drylake in the dark alley, still visible in very light conditions too.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1acfe017e1ff2d4c9fe7fd7982ba7e2159106ee9 available at: https://gist.github.com/e3a96895a79bd102f0ead62e57e943d8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8615-1acfe01.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8615-1acfe01.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2ddd46f30b167497fa416604335362ce3514c07c available at: https://gist.github.com/e3a96895a79bd102f0ead62e57e943d8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8615-2ddd46f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8615-2ddd46f.dmg

File: scripts/system/controllers/handControllerGrab.js [hunk 1]
```diff
@@ -602,46 +604,24 @@ EquipHotspotBuddy.prototype.updateHotspot = function(hotspot, timestamp) {
             targetSize: 1,
             overlays: []
         };
-
+
```
Comment (hyperlogic) [inline scripts/system/controllers/handControllerGrab.js]: dangling whitespace

**Comment by hifi-gustavo** [comment]
Build console output for commit: 22d80914e2d69614a72904c47720af26e4e32486 available at: https://gist.github.com/e3a96895a79bd102f0ead62e57e943d8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8615-22d8091.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8615-22d8091.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ade17e6456841acbf113ec5e7929814cfd34e943 available at: https://gist.github.com/e3a96895a79bd102f0ead62e57e943d8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8615-ade17e6.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8615-ade17e6.dmg

**Comment by davidkelly** [comment]
## Test Plan:

### Check it in dark places
Go to somewhere dark -- alley in Drylake for instance.  Equip the ping pong ball gun, (load the `tutorials/createPingPongBallGun.js`), ensure you can see it well enough.  

### Check it in light places
Then go somewhere very bright (maybe welcome room or similar), and do the same.

**Comment by RyanDowne** [comment]
There are some color combos that aren't right, like if you go to Eschatology-dev you'll see the purple geo dome doesn't play that well with the green floor. 

I think this is an improvement though so will call it a pass.

**Pull request merged**